src/io/flutter/run/bazel/FlutterBazelConfigurationEditorForm.form

```diff
@@ -111,10 +111,37 @@
       </component>
       <component id="ada9e" class="javax.swing.JTextField" binding="myAdditionalArgs">
         <constraints>
-          <grid row="6" column="1" row-span="1" col-span="1" vsize-policy="0" hsize-policy="6" anchor="0" fill="1" indent="0" use-parent-layout="false"/>
+          <grid row="8" column="1" row-span="1" col-span="1" vsize-policy="0" hsize-policy="6" anchor="0" fill="1" indent="0" use-parent-layout="false"/>
         </constraints>
         <properties/>
       </component>
+      <component id="5a54b" class="javax.swing.JCheckBox" binding="myEnableReleaseModeCheckBox" default-binding="true">
+        <constraints>
+          <grid row="6" column="1" row-span="1" col-span="1" vsize-policy="0" hsize-policy="3" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <enabled value="true"/>
+          <selected value="false"/>
+          <text value="Enable release mode, --define flutter_build_mode=release"/>
+        </properties>
+      </component>
+      <component id="f7b9" class="javax.swing.JLabel">
+        <constraints>
+          <grid row="6" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <text value="&amp;Release mode:"/>
+        </properties>
+      </component>
+      <component id="66407" class="javax.swing.JLabel">
+        <constraints>
+          <grid row="7" column="1" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <enabled value="false"/>
+          <text value="When enabled, debugging will no longer work; and a physical device will be required when launching."/>
```

Comment:
Perhaps: `This mode does not support debugging, and a physical device will be required when launching.`

Reply:
Done.